source/os/operations_win.c

```diff
@@ -317,6 +319,32 @@ bool minibat(const struct args args) {
     return true;
 }
 
+bool add_path(const os_char *new_path) {
+    CHAR buffer[SHRT_MAX];
+    DWORD bufferSize = sizeof(buffer) / sizeof(CHAR);
+    
+    if (!get_shell_env("PATH", bufferSize, buffer)) {
+        format_error("Fail to get PATH environment variable!\n");
+        return false;
+    }
+    
+    if (strstr(buffer, new_path) != NULL) {
+        format_output("The path %s is already in the PATH environment variable.\n", new_path);
```

Comment:
Sai trong trường hợp muốn thêm 
`/path/to/dir/subdir`
mà PATH đã có 
`/path/to/dir`.
T nghĩ check thêm kí tự tiếp theo có phải `;` (`:`) không là oke.